.agent/skills/drn-utils/SKILL.md

```
@@ -182,400 +100,204 @@ public class FeatureFlags { }
-private MyService? _instance;
-LockUtils.TrySetIfNull(ref _instance, new MyService());
 ```
 
 ---
 
-## ScopeContext (Ambient Context)
+## Utilities Reference
 
-Access scoped data anywhere. Simplifies cross-cutting concerns like auditing, multi-tenancy, and security by avoiding deep parameter passing, especially in Razor Pages (`.cshtml`).
+| Area | Key Types | Purpose |
+|------|-----------|---------| 
```

markdownlint-cli2 (0.20.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)